src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -423,6 +441,28 @@ export default class BrowserCell extends Component {
       classes.push(styles.required);
     }
 
+    let extras = null;
+    if (this.state.showConfirmationDialog)
+      extras = (
+        <Modal
+          type={this.selectedScript.confirmationDialogStyle === 'info' ? Modal.Types.INFO : Modal.Types.DANGER}
```

Comment:
```suggestion
          type={this.selectedScript.confirmationDialogStyle === 'critical' ? Modal.Types.DANGER : Modal.Types.INFO}
```

Changing the default style to info.